zebra-chain/src/work/difficulty.rs

```diff
@@ -112,6 +112,16 @@ impl fmt::Debug for ExpandedDifficulty {
     }
 }
 
+impl fmt::Display for ExpandedDifficulty {
```

Comment:
Should this be feature flagged?

Reply:
https://github.com/ZcashFoundation/zebra/pull/5659/commits/d30e5e4143063d685f140697a08c0f95cf429147